wagtail/admin/templates/wagtailadmin/pages/editor_sidebar_panels/status.html

```diff
@@ -0,0 +1,30 @@
+{% load wagtailadmin_tags i18n %}
+
+<div class="w-divide-y w-divide-grey-100 w-sticky w-top-0">
```

Comment:
Does this still need to be sticky? Now looks like the whole drawer is fixed.

```suggestion
<div class="w-divide-y w-divide-grey-100">
```